auxiliary/notes.tex

```diff
@@ -4,11 +4,42 @@
 This version of the book has been substantially revised
 since the first release in June 2019
 with feedback from readers and other experts.
-It now contains most of the major content
+We especially want to recognize the contributions of
```

Comment:
perhaps 'thank' instead of 'recognize the contributions of'? i would say in addition to, but the sentence is already very long/complex... but i think we want to express thanks/gratitude instead of only recognition